patroni/ha.py

```diff
@@ -1168,6 +1168,8 @@ def shutdown(self):
             if not self.state_handler.is_running():
                 if self.has_lock():
                     self.dcs.delete_leader()
+                self.state_handler.set_state('stopped')
```

Comment:
The state should already be `stopped` at this moment: https://github.com/zalando/patroni/blob/master/patroni/postgresql.py#L959

Reply:
I did not experience the state changing from running to anything else.  It simply stayed as "running" and eventually, the etcd entry timed 
![patroni_stopped_state](https://user-images.githubusercontent.com/4976173/42816892-e5b85e30-8991-11e8-8581-0467eb0c203c.png)
out and was deleted.  See attached image for an example.

Reply:
I did not experience the "stopped" status being recorded.  The only status I saw was "running" until the etcd entry timed out.  The attached image shows the output from patronictl as the member was running, two after it stoped, and a final once the etcd entry timed out.

![patroni_stopped_state](https://user-images.githubusercontent.com/4976173/42816942-1bee25fc-8992-11e8-9954-14d90daf9ef3.png)

Reply:
I just wanted to tell that it would be enough to call `touch_member`, without calling `self.state_handler.set_state('stopped')`

Reply:
Ah.... got it.  I'll try that so there is one less call.  If that works, can the change be accepted?